lib/maintenance_mode_message.js

```diff
@@ -14,10 +14,19 @@ module.exports = {
  * require('aws-sdk/lib/maintenance_mode_message').suppress = true;
  */
 function emitWarning() {
+  if (typeof process !== 'undefined')
```

Comment:
Check for process being undefined
```suggestion
  if (typeof process === 'undefined')
```